go.mod

```diff
@@ -426,7 +426,10 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-require github.com/DataDog/go-libddwaf v0.0.0-20221118110754-0372d7c76b8a
+require (
+	github.com/DataDog/go-libddwaf v0.0.0-20221118110754-0372d7c76b8a
+	go.opentelemetry.io/collector/confmap v0.68.0
```

Comment:
Can you move this to the main require section alongside the other `go.opentelemetry.io/collector/*` dependencies?